app/controllers/admin/affiliates_controller.rb

```diff
@@ -66,7 +66,38 @@ class Admin::AffiliatesController < Admin::AdminController
       config.columns[c].form_ui = :textarea
     end
 
-    update_columns = attribute_columns.reject { |column| column =~ /\A(api_access_key|created_at|external_css_url|favicon_url|has_staged_content|id|nutshell_id|theme|updated_at)\z/i }
+    update_columns = %i[
+                         active
+                         agency
+                         bing_v5_key
+                         dap_enabled
+                         display_name
+                         domain_control_validation_code
+                         fetch_concurrency
+                         force_mobile_format
+                         ga_web_property_id
+                         gets_blended_results
+                         gets_commercial_results_on_blended_search
+                         gets_i14y_results
+                         google_cx
+                         google_key
+                         i14y_date_stamp_enabled
+                         is_bing_image_search_enabled
+                         is_federal_register_document_govbox_enabled
+                         is_medline_govbox_enabled
+                         is_photo_govbox_enabled
+                         is_related_searches_enabled
+                         is_rss_govbox_enabled
+                         is_sayt_enabled
+                         is_video_govbox_enabled
+                         jobs_enabled
+                         locale
+                         name
+                         raw_log_access_enabled
+                         search_consumer_search_enabled
+                         search_engine
+                         website
+                       ]
     config.update.columns = []
```

Comment:
Possibly. However, I find it helpful to have them visible in the controller itself, because the various column lists in that controller are added and subtracted from each other. Moving all those lists to a separate initializer could make the code flow more difficult to read. That file could certainly use some cleanup in general, but that should probably be done as a separate story.

Reply:
Its best practice to have skinny controllers. Actually now days even skinny models.  https://medium.com/marmolabs/skinny-models-skinny-controllers-fat-services-e04cfe2d6ae. You can look into it as a separate cleanup story if you want. But certainly makes it harder to read with all that clutter.  :).  Why are these columns being added and subtracted?

Reply:
The addition and subtraction is just how that controller was originally written. The various column sets (used for displaying the various affiliate settings in super admin) are put together with logic such as "foo_colums = bar_columns + baz_columns", or "foo columns = bar_columns - baz_columns". Before doing any heavy refactoring on that controller, we'd probably want to audit the affiliates table in general to determine which columns can be completely dropped (as many of those support legacy features).